ipynb/vioscreen_comparisons.ipynb

```diff
@@ -0,0 +1,1009 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import skbio\n",
+    "import scipy as sp\n",
+    "import seaborn as sns\n",
+    "from emperor import Emperor, nbinstall\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.gridspec as gridspec\n",
+    "from matplotlib.backends.backend_pdf import PdfPages\n",
+    "% matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mapping = pd.read_csv('viscreen_subsample_mapping.txt', sep='\\t') "
```

Comment:
Yes, Justine. You are right. I also include the detailed code for filtering in the new notebook.